### PR TITLE
FSE: add theme activation notice

### LINKED ISSUE #61119: FSE: update the messaging during FSE theme activation
This is what we currently show when someone switches to an FSE theme:

screenshot-2022-02-11-at-15 50 13-1-edited

@poligilad-auto proposed that we add the following text here:

> This theme has full site editing. Now you can design everything on your site, from the header right down to the footer, using blocks.

You can check out the discussion in this thread for more context: paYE8P-1rO-p2

### PR DESCRIPTION
**Pull request by zaguiini**
Fixes #61119.

#### Changes proposed in this Pull Request

We want to display a message indicating the capabilities of FSE upon theme activation.

image

#### Testing instructions

- Open Themes page;
- Activate one FSE theme;
- Check the notice in the Thanks modal.

### CODE REVIEW

File: client/my-sites/themes/thanks-modal.scss [hunk 1]
```diff
@@ -53,10 +53,11 @@
 		}
 	}
 
-	.thanks-modal__gutenberg-prompt {
+	&-fse-notice {
+		margin: 1rem auto;
 		color: var( --color-text-subtle );
-		font-size: 14px;
-		margin-top: 12px;
+		font-size: 0.875rem;
```
Comment (jeyip) [inline client/my-sites/themes/thanks-modal.scss]: Could we use a font prescribed by `packages/typography/styles/variables.scss` (which already appears to use rem units)?

https://github.com/Automattic/wp-calypso/blob/a67e5be134d785af8dbd2612270f4290f36f662d/packages/typography/styles/variables.scss#L27-L48

**Comment by jeyip** [comment]
## Testing
### Requirements
#### Simple
- [x] Thanks modal displays updated FSE copy when activating an FSE theme
- [x] Thanks modal does not displays FSE copy when activating a classic theme

#### Atomic
- [ ] Thanks modal displays updated FSE copy when activating an FSE theme
- [ ] Thanks modal does not displays FSE copy when activating a classic theme

### Browsers
- [ ] Edge
- [ ] Firefox
- [ ] Safari
- [x] Chrome

The copy doesn't appear to be updated for my atomic site @zaguiini 

Screen Shot 2022-03-02 at 2 32 40 PM

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7163545

Thank you @zaguiini for including a screenshot in the description! This is really helpful for our translators.

**Comment by zaguiini** [comment]
>The copy doesn't appear to be updated for my atomic site

That's weird, @jeyip. I noticed in your screenshot that it also doesn't say `Edit Site` in the CTA, so I wonder if that's related to my PR or something related to FSE itself 🤔

**Comment by jeyip** [comment]
> I noticed in your screenshot that it also doesn't say Edit Site in the CTA, so I wonder if that's related to my PR or something related to FSE itself 🤔

That's a good observation. I'm not sure, but I can help take a look either later today or tomorrow.

**Comment by jeyip** [comment]
@zaguiini 

I did some digging and you're right. `isFSEActive` looks like it returns stale data on atomic sites for some period of time. p1646421378255859-slack-CHN6J22MP

**Review by jeyip — COMMENTED** [review]
Based off of discussions in p1646421378255859-slack-CHN6J22MP, it might be preferable to display the FSE activation notice by evaluating theme tags, theme templates, etc. instead of relying on block editor settings.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7163545).

**Comment by jeyip** [comment]
## Testing
### Requirements
#### Simple
- [x] A message is displayed indicating the capabilities of FSE upon FSE theme activation
- [x] No message is displayed when a classic theme is activated

#### Atomic
- [x] A message is displayed indicating the capabilities of FSE upon FSE theme activation
- [x] No message is displayed when a classic theme is activated

### Browsers
- [x] Edge
- [x] Firefox
- [x] Chrome

**Review by jeyip — APPROVED** [review]
Lgtm!

**Pull request merged**